x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/components/curation_queries/curation_query.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiFlexGroup, EuiFlexItem, EuiFieldText, EuiButtonIcon } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import './curation_queries.scss';
+
+interface Props {
+  queryValue: string;
+  onChange(newValue: string): void;
+  onDelete(): void;
+  disableDelete: boolean;
+}
+
+export const CurationQuery: React.FC<Props> = ({
+  queryValue,
+  onChange,
+  onDelete,
+  disableDelete,
+}) => (
+  <EuiFlexGroup className="curationQueryRow" alignItems="center" responsive={false} gutterSize="s">
+    <EuiFlexItem>
+      <EuiFieldText
+        fullWidth
+        placeholder={i18n.translate(
+          'xpack.enterpriseSearch.appSearch.engine.curations.queryPlaceholder',
+          { defaultMessage: 'Enter a query' }
+        )}
+        value={queryValue}
+        onChange={(e) => onChange(e.target.value)}
+      />
+    </EuiFlexItem>
+    <EuiFlexItem grow={false}>
+      <EuiButtonIcon
+        iconType="trash"
+        color="danger"
+        onClick={onDelete}
+        isDisabled={disableDelete}
+        aria-label={i18n.translate('xpack.enterpriseSearch.actions.delete', {
```

Comment:
This is a pretty generic name, is there a risk of collision with other places where this is used? 

> xpack.enterpriseSearch.actions.delete

Reply:
This was intentional, at some point (maybe next week if I have time?) I'd like to DRY out these really small actions/button labels like "Edit", "Delete", "Save", "Cancel", "Continue" etc. strings so we just reuse the same const/string and don't make localizers translate them repeatedly.